### DISCUSSION TITLE
aggregate query pagination  not work in mongodb

### DISCUSSION THREAD

**Discussion by arjundexbytes** [Power-Components/livewire-powergrid#1378] [General]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

No

### PHP Version

8.3.1

### PowerGrid

5.3

### Laravel

10.40

### Livewire

3.3.5

### Alpine JS

3.10.3

### Theme

Tailwind 3.x, Tailwind 3.x with tailwind/forms

### Describe the bug.

it's working,but query not excute working on collection data not dynamic data,in that case page is working very slow if data count is high 

### To Reproduce...

First click on "FOO" then....

### Extra information

```markdown
<?php
 //...
```

**Comment by dansysanalyst** [comment]
Hi @arjundexbytes,

There are no plans to integrate MongoDB as it is not currently listed in Laravel's [first-party support databases](https://laravel.com/docs/10.x/database#introduction).

Having that said, you are welcome to submit a Pull Request adding/improving any features you consider valuable. 

All contributions are appreciated!
